### PR TITLE
Tree from object

### PR DESCRIPTION
**Pull request by gestap**
added new task, example img, updated readme
updated package.json, add new dependencies and scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,49 @@
-# Base template for Gulp
+# Tree from object
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Write a function `createTree` that creating list of `ul/li` from object.
```
Comment (roman-mate) [inline README.md]:
```suggestion
Write a function `createTree` that creates a list of `ul/li` from an object.
```

File: README.md [hunk 2]
```diff
@@ -1,65 +1,49 @@
...
+`element` - is a DOM element.
 
-## Development mode 
-
-```bash
-$ npm run start
-```
-
-## Build the project
+## Workflow
```
Comment (roman-mate) [inline README.md]: We should extract this repetitive workflow into one guidelines file.

Comment (gestap) [reply README.md]: Created new repo - https://github.com/mate-academy/js_task-DOM-guideline
removed workflow and add a link to guideline

File: README.md (hunk 1 above)
Comment (gestap) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1,79 +1,21 @@
-# Base template for Gulp
+# Tree from object
 
-Template to start from before starting frontend projects.
+## Task
 
-## Install template
+Write a function `createTree` that creates a list of `ul/li` from an object.
```
Comment (mgrinko) [inline README.md]: `createTree(element, data)` - to avoid the next line
`a nested list`

Comment (gestap) [reply README.md]: changed

**Comment by gestap** [comment]
@roman-mate, @mgrinko, @ivadimko, @vpolets 
I need your approval or comments

**Pull request merged**